lib/ash.ex

```diff
@@ -1973,15 +1977,18 @@ defmodule Ash do
   """
   @doc spark_opts: [{1, @create_opts_schema}]
   @spec create(
-          changeset_or_resource :: Ash.Changeset.t() | Ash.Resource.t(),
+          changset_or_resource :: Ash.Changeset.t() | Ash.Changeset.t(),
+          params_or_opts :: map() | Keyword.t(),
           opts :: Keyword.t()
         ) ::
           {:ok, Ash.Resource.record()}
           | {:ok, Ash.Resource.record(), list(Ash.Notifier.Notification.t())}
           | {:error, term}
-  def create(changeset_or_resource, opts \\ [])
+  def create(changeset_or_resource, params_or_opts \\ %{}, opts \\ [])
+
+  def create(%Ash.Changeset{} = changeset, params_or_opts, opts) do
+    {_params, opts} = Ash.Helpers.get_params_and_opts(params_or_opts, opts)
```

Comment:
🤔 I think we should do this kind of thing here:

```elixir
if changeset.__validated_for_action__ && params != %{} do
  Logger.warn("input is being provided for a changeset that has already been validated for an action. params are being ignored"
else
  Ash.Changeset.for_create(changeset, params, ....)
end
```

Reply:
Or maybe raise an error?

I see no case where we could call `Ash.create(changeset, params, opts)`.

Reply:
We can raise an error, sure 👍 but we should handle that second case, where the changeset has not yet been validated for an action. That way you can
```elixir
Resource
|> Ash.Changeset.new()
|> Ash.Changeset.set_context(...)
|> Ash.create!(%{foo: :bar})
```

Reply:
561574b should solve this issue.